Assets/MixedRealityToolkit/_Core/Definitions/MixedRealityConfigurationProfile.cs

```diff
@@ -202,20 +202,16 @@ public SystemType BoundarySystemSystemType
         }
 
         [SerializeField]
-        [Tooltip("Instruct the platform whether or not to render the playspace boundary. Note: not all platforms support configuring this option.")]
-        private bool enablePlatformBoundaryRendering = false;
+        [Tooltip("The approximate height of the play space, in meters.")]
+        private float boundaryHeight = 3.0f;
 
         /// <summary>
-        /// Instruct the platform whether or not to render the playspace boundary.
+        /// The approximate height of the play space, in meters.
```

Comment:
approximate?

Reply:
It's a developer estimate, there is no measurement. I'll re-phrase the comment.